src/xSTIR/pSTIR/STIR.py.in

```diff
@@ -840,6 +840,16 @@ class AcquisitionData(DataContainer):
         info = pyiutil.charDataFromHandle(handle)
         pyiutil.deleteDataHandle(handle)
         return info
+    def convert_to_image(self):
+        """Convert sinogram to image. This might be useful
+        for visualising with an external viewer. Geometrical
+        information is set arbitrarily."""
+        sino_arr = self.as_array()[1:]
```

Comment:
why this? It seems even dangerous as `ImageData.fill` should ideally do some consistency checks on dimensions.

Reply:
Do you mean why the `[1:]`? Well, because sinograms are currently 4D (with 0th index as singleton), whereas are easier to view as 3D.